frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -169,16 +169,36 @@ export const getServerlessData = (resource: K8sResourceKind) => {
       maxpods: '',
       concurrencytarget: '',
       concurrencylimit: '',
+      autoscale: {
+        autoscalewindow: '',
+        autoscalewindowUnit: '',
+        defaultAutoscalewindowUnit: '',
+      },
+      concurrencyutilization: '',
     },
   };
+  const windowRegEx = /^[0-9]*|[a-zA-Z]*/g;
   if (getResourcesType(resource) === Resources.KnativeService) {
     const annotations = _.get(resource, 'spec.template.metadata.annotations');
+    const autoscalewindow = _.get(annotations, 'autoscaling.knative.dev/window', '').match(
```

Comment:
i know we have usage of `lodash` here can avoid here and good to move `autoscaling.knative.dev` to const  as used in other places as well

Reply:
+1